cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -177,33 +178,50 @@ mkTracers traceOptions tracer = Tracers
           case ev' of
               (ChainDB.TraceAddBlockEvent ev) -> case ev of
                   ChainDB.SwitchedToChain _ c -> do
-                    case withOriginToMaybe . pointSlot $ AF.headPoint c of
-                      Nothing -> pure () -- We are at genesis block.
-                      Just slotNum -> do let sNum :: Integer
-                                             sNum = fromIntegral $ unSlotNo slotNum
-                                             logValue :: LOContent a
-                                             logValue = LogValue "slotNum" . PureI $ sNum
-                                         -- phash = fst tippair
-                                         meta <- mkLOMeta Debug Confidential
-                                         let tr' = appendName "slotNum" tr
-                                         traceNamedObject tr' (meta, logValue)
+                      meta <- mkLOMeta Critical Confidential
+                      let tr' = appendName "metrics" tr
+                          chaininfo = chainInformation c
+                          sNum = slots chaininfo
+                          bNum = blocks chaininfo
+                          epochSlots :: Word64 = 21600  -- TODO
```

Comment:
Lets file an issue for this, and also for the chain density measure so we do not forget. Both of these are things where the details of the calculation will depend on the consensus implementation.

In Praos there are 10x more slots than blocks (in expectation).

For density, the way that is calculated will also change across the hard fork boundary, so ought to be provided by the consensus, since only it really knows which consensus algorithm we are using, and when it changes as we cross hard fork boundaries during the program run.

Reply:
issue: https://github.com/input-output-hk/cardano-node/issues/272